.github/workflows/changelog.yml

```diff
@@ -0,0 +1,8 @@
+on: pull_request
```

Comment:
Also, I understand we don't need a changelog for every contribution. So, should we add rules to the github action so as to when to run this workflow?

```yml
on:
  pull_request:
    paths:
      - 'grimoirelab_toolkit/'
```

https://github.com/chaoss/grimoirelab-toolkit/blob/master/CONTRIBUTING.md#what-warrants-a-changelog-entry
https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#onpushpull_requestpaths

This would restrict the workflow on any contribution unless it is a change to the `grimoirelab_toolkit` source code.

Anyway, this is just an idea. We can enable the workflow on all the pull requests and choose to ignore this workflow if the PR doesn't need a changelog file.

Please let me know what do you think?

Reply:
Probably that's a good way to improve the work for the reviewers. We can try it!

Reply:
Okay